### ISSUE TITLE
Reproducing problem

### ISSUE THREAD

**Issue by LingyiHongfd** [CODE4UVOS/HFAN#4]
Hello, HFAN is an interesting work and thanks for your open-resourcing. When I reproduce the HFAN by using the same setting as yours, I meet with some problems. I just clone the code, download the data provided by you, and change the path. I cannot reproduce the result. Would you mind sharing the training log or the checkpoint after pretraining YouTube-VOS? Thanks a lot.

**Comment by Anonymous-SSVOS** [comment]
Thank you for your interest.

Could you list your current quantitative results?

**Comment by CODE4UVOS** [comment]
Thank you for your interest.

Could you list your current quantitative results?

**Comment by LingyiHongfd** [comment]
Sorry, I have reproduced the results. Because I modified the priority of eval hook, so I can eval the model during training. I found the performance on davis eval fluctuates sharply between 65 and 75 during training, although the loss is converging. HFAN pretrained on YTB just achieves about 73 J and F. But after finetuning on DAVIS, I successfully reproduced the result. Because you didn't open resource the pretraining result and log, I initially saw this drastic fluctuation and suspected that the results could not be reproduced. Thanks for your replay and this interesting work.
